modules/ROOT/nav.adoc

```diff
@@ -386,6 +386,7 @@ include::analytics:partial$nav.adoc[]
  ** xref:install:upgrade-offline.adoc[Performing the Offline Upgrade]
  ** xref:install:upgrade-individual-nodes.adoc[Performing the Single Node Upgrade]
  ** xref:install:upgrade-to-ipv6.adoc[Upgrading IPv6 Clusters]
+ ** xref:install:upgrade-strategy-for-features.adoc[Upgrading to Couchbase Server 6.5]
```

Comment:
Should this be 
xref:install:upgrade-strategies.adoc[Upgrading to Couchbase Server 6.5]
?

Reply:
no, this is a new file just for features

Reply:
In that case, the file I named needs its page-alias removed.
Is that in another PR?

Reply:
ah, it's commented out

Reply:
confirmed that it's commented already